src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -1365,16 +1365,19 @@ void func_80B45648(EnInvadepoh* this) {
     }
 }
 
-s32 func_80B456A8(PlayState* play, Vec3f* vec) {
-    Vec3f multDest;
-    f32 wDest;
-
-    Actor_GetProjectedPos(play, vec, &multDest, &wDest);
-    if (((multDest.z > 1.0f) && (fabsf(multDest.x * wDest) < 1.0f)) && (fabsf(multDest.y * wDest) < 1.0f)) {
-        s32 wX = (multDest.x * wDest * 160.0f) + 160.0f;
-        s32 wY = (multDest.y * wDest * -120.0f) + 120.0f;
-        s32 wZ = (s32)(multDest.z * wDest * 16352.0f) + 0x3FE0;
-        s32 zBuf = func_80178A94(wX, wY);
+s32 func_80B456A8(PlayState* play, Vec3f* worldPos) {
```

Comment:
You could probably name this, looks like it's almost a re-implementation of that other function.

Reply:
I'm not exactly following what this function is doing, particularly the `s32 zBuf = func_80178A94(screenPosX, screenPosY);`

The other function, `Lights_GlowCheck` is checking for lights, but there are no lights in this function. It's probably checking some distance or within some proximity, but I'm not 100% sure to name it